docs: add webhook example (#7565)
langflow-ai/langflow: docs/docs/Develop/webhook.md

---
title: Trigger flows with webhooks
slug: /webhook
---

import Icon from "@site/src/components/icon";

> To enable your flow to be triggered by an external request, add a **Webhook** component to your flow.

Author: ```suggestion
Add a **Webhook** component to your flow to trigger it with external requests. 
```
